src/main/java/domain/Date.java

```diff
@@ -0,0 +1,59 @@
+package domain;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+public class Date {
+
+    private static final int REQUIREMENT_YEAR = 2024;
+    private static final int REQUIREMENT_MONTH = 12;
+    private static final List<Integer> HOLIDAY = List.of(25);
+
+    private LocalDate localDate;
```

Comment:
```suggestion
    private final LocalDate localDate;
```
final로 선언해도 될 것 같아요.

Reply:
아!! 감사합니다 수정하였습니다